### PR TITLE
Component tooltip for personality

### PR DESCRIPTION
**Pull request by Toxin4**
Tooltip component created stating that personality information is taken from Wikidata
New design:
Captura de tela 2024-05-02 150153
image
image
image

### CODE REVIEW

**Comment by Toxin4** [comment]
The hide personality component was removed and added to the tooltip component, now the tooltip component does both. And the translation into Portuguese and English has been added.
Viewing changes made:
Captura de tela 2024-05-02 121554
Captura de tela 2024-05-02 121715
image

File: src/components/Personality/PersonalityCardAvatar.tsx [hunk 1]
```diff
@@ -25,32 +25,22 @@ const PersonalityCardAvatar = ({
...
-                    <AletheiaAvatar
-                        size={componentStyle.avatarSize}
-                        src={personality.avatar}
-                        alt={t("seo:personalityImageAlt", {
-                            name: personality.name,
-                        })}
-                    />
-                </HiddenPersonalityAvatarTooltip>
-            )}
-            {!hoistAvatar && !personality.isHidden && (
```
Comment (pepermao) [inline src/components/Personality/PersonalityCardAvatar.tsx:45]: This conditional should not be deleted, it should involve the component, e.g:
```js
{!hoistAvatar && !personality.isHidden && (
     <PersonalityCardTooltip
          isHidden={personality?.isHidden}
          offset={tooltipOffset}
          style={{
              padding: componentStyle.hiddenIconSize / 2.5,
              fontSize: componentStyle.hiddenIconSize,
          }}
      >
)}

File: src/components/Personality/PersonalityCardTooltip.tsx [hunk 2]
```diff
@@ -1,20 +1,30 @@
+import React from "react";
```
Comment (pepermao) [inline src/components/Personality/PersonalityCardTooltip.tsx:1]: Rename the file to `PersonalityCardAvatarTooltip.tsx`

File: src/components/Personality/PersonalityCardTooltip.tsx [hunk 3]
```diff
@@ -1,20 +1,30 @@
...
 
-const HiddenPersonalityAvatarTooltip = ({ children, offset, style }) => {
+const PersonalityCardTooltip = ({ children, isHidden, style, offset }) => {
     const { t } = useTranslation();
 
     return (
         <Popover
-            placement="topRight"
-            content={t("personality:hiddenPersonalityAvatarTooltip")}
+            placement="right"
```
Comment (pepermao) [inline src/components/Personality/PersonalityCardTooltip.tsx:12]: change placement to `topRight`

**Review by pepermao — CHANGES_REQUESTED** [review]

**Review by pepermao — APPROVED** [review]

**Review by thesocialdev — APPROVED** [review]

**Pull request merged**